randovania/games/dread/json_data/Cataris.txt

```diff
@@ -1182,10 +1184,10 @@ Extra - asset_id: collision_camera_015
       Any of the following:
           All of the following:
               Varia Suit
-              Spider Magnet or Space Jump or Simple IBJ
+              Spider Magnet or Simple IBJ or Use Spin Boost
           All of the following:
               Heat/Cold Runs (Intermediate) and Heat Damage ≥ 200
-              Spider Magnet or Space Jump
+              Spider Magnet or Use Spin Boost
```

Comment:
Did this entire section wrong. The connection should start as an All of the following, then an Or containing Varia or the heat run, then another Or containing Spider, IBJ, or spin boost. Furthermore, if using just spin boost i feel it should use Wall Jump (Beginner cause you have to wall jump back into the wall after the spin boost i think. I'd need to test it again. If you only need a single wall jump then it's fine as is.

Reply:
I didn't set it up the way it was but it has been overhauled, it was split like it was to give more heat damage for the IBJ which I maintained when overhauling it so every instance of Varia is in an Or group with the damage run trick and resource